packages/blob/src/api.node.test.ts

```diff
@@ -51,22 +51,22 @@ describe('api', () => {
       );
 
       const res = await requestApi<{ success: boolean }>(
-        '/api',
+        '/method',
         { method: 'POST', body: JSON.stringify({ foo: 'bar' }) },
         { token: '123' },
       );
 
       expect(fetchMock).toHaveBeenCalledTimes(1);
       expect(fetchMock).toHaveBeenNthCalledWith(
         1,
-        'https://blob.vercel-storage.com/api',
+        'https://vercel.com/api/blob/method',
         {
           body: '{"foo":"bar"}',
           headers: {
             authorization: 'Bearer 123',
             'x-api-blob-request-attempt': '0',
             'x-api-blob-request-id': expect.any(String) as string,
-            'x-api-version': '10',
+            'x-api-version': '11',
```

Comment:
why is this an api version change?

Reply:
I want to ensure we can track down this update in metrics (i.e. compare retries by version for example)